webpack.config.js

```diff
@@ -68,7 +68,7 @@ module.exports = {
   module: {
     rules: [
       {
-        test: /node_modules\/((ansi-styles|strip-ansi|ansi-regex|react-dev-utils|chalk|typescript-plugin-inner-jsx)\/).*/,
+        test: /node_modules\\((ansi-styles|strip-ansi|ansi-regex|react-dev-utils|chalk|typescript-plugin-inner-jsx)\\).*/,
```

Comment:
Why did you change a regex? It's incorrect now.

Reply:
No, it's correct, now. https://docs.microsoft.com/en-us/dotnet/standard/io/file-path-formats